cmd/fyne_demo/screens/container.go

```diff
@@ -0,0 +1,109 @@
+package screens
+
+import (
+	"fmt"
+	"image/color"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+)
+
+// ContainerScreen loads a tab panel for containers and layouts
+func ContainerScreen() fyne.CanvasObject {
+	return widget.NewTabContainer(
+		widget.NewTabItem("Split", makeSplitTab()),
+		widget.NewTabItem("Scroll", makeScrollTab()),
```

Comment:
Tabber is missing from this list, but since the whole thing is in a tabber I think it's ok